### DISCUSSION TITLE
play drag and dropped video links

### DISCUSSION THREAD

**Discussion by marten27** [qnblackcat/uYouPlus#689] [Ideas]
since i'm currently logged out from my youtube account in uYouPlus, i'd love to the ability 
to drag and drop video links (by dragging thumbnail and title) from the Youtube app into uYouPlus, so the videos open automatically.
i've noticed i only get the video link when i drag and drop thumbnail and title from my subscsriptions feed or from my search results, weirdly not though from recommended videos. but that is mighty fine by me (-:

**Comment by bgy78uhn** [comment]
I second this. When I'm on my ipad I like to use both apps side by side, watching videos fullscreen on uYouPlus while browsing on Youtube. If i could drag and drop those video tiles (i guess thats what the  thumbnail and title cards are called) from Youtube to uYouPlus and then the video automatically opens like it does in Safari, that would be a great benefit to me as well. Like with Safari, it probably would be also possible to just drag the video tile on to the uYouPlus app symbol in your dock.
